preview/Win7Msix/Win7MSIXInstaller/MsixResponse.hpp

```diff
@@ -0,0 +1,52 @@
+#pragma once
+#include <windows.h>
+#include <string>
+#include "GeneralUtil.hpp"
+
+class MsixResponse

```

Comment:
Wouldn't progress also go on this interface? We should add comments here to then as we merge with Rudy's change we can make sure it is aligned.

Reply:
Yes, added comments for progress bar changes.